psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -3122,7 +3276,8 @@ void FISAPT::felst() {
     std::vector<double> Elst10_terms;
     Elst10_terms.resize(4);
 
-    matrices_["Elst_AB"] = std::make_shared<Matrix>("Elst_AB", nA + na, nB + nb);
+    matrices_["Elst_AB"] = std::make_shared<Matrix>("Elst_AB", nA + na + 1, nB + nb + 1); // Add one entry for external
```

Comment:
Are these matrices defined anywhere? If you're changing the matrix dimensions, I would think you're changing its definition, which worries me.

Reply:
As mentioned above, I now have a comment that describe this matrix.